hs_core/hydroshare/utils.py

```diff
@@ -442,6 +445,39 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id):
         tgt_res.resources.set(src_res.resources.all())
 
 
+@sync_to_async
+def _get_relations():
+    return list(GeospatialRelation.objects.all())
+
+
+@sync_to_async
+def _save_relation(relation, json):
+    return relation.update_from_geoconnex_response(json)
+
+
+async def get_jsonld_from_geoconnex(relation, client):
+    relative_id = relation.value.split("ref/").pop()
+    collection = relative_id.split("/")[0]
+    id = relative_id.split("/")[1]
+    url = f"/collections/{collection}/items/{id}?" \
+               "f=jsonld&lang=en-US&skipGeometry=true"
+    print(f"CHECKING RELATION '{relation.text}'")
```

Comment:
Very minor, but there are two print statements that should be updated to a logger.  Generally try to keep print statements out of the codebase with the exception of commands. Since this is used in a celery task, it should a logger.

Reply:
Thanks @sblack-usu, good catch!